RELEASE.md

```diff
@@ -0,0 +1,63 @@
+# Release
+
+The processs of creating a release is as follows.
+
+## Schedule
+
+Releases are made on an ad-hoc basis currently, rather than on an
+elapsed time basis.  Typically once there are significant changes.
+
+## Invariants / rules
+
+- Pull Requests should not change the version number in `project.clj`.
+- Each Pull Request should update the `CHANGELOG.md` file as
+  appropriate.
+- In the normal state of this repository, the version in `project.clj`
+  should be of the form `x.y.z-SNAPSHOT`, denoting that the repo is in
+  between versions.
+
+## Release preconditions
+
+- Your local repo should be clean.
+- All tests pass.
+
+## Release process
+
+Note this uses `0.1.12` as an example new release number.
+
+1. Create a release branch on your local repo of your fork of this
+  repo on Github.
+  ```
+  git checkout master
+  git pull
+  git checkout -b release-0.1.12
+  git push -u origin release-0.1.12
+  ```
+2. Update [CHANGELOG.md](CHANGELOG.md)
+  * Write the release highlights in the `Unreleased` section.
+  * Add a section for the new release just under the `Unreleased` section.
+  * Update the links section at the end of the file to include links
+    for the new release and update the link for the Unreleased
+    version.
+3. Update the version in [project.clj](project.clj). 
+4. Update any version numbers in [README.md](README.md).
+5. Push changes to your fork on Github.
```

Comment:
I do not see why we would need to require the use of a fork

Reply:
I'm not sure I like the idea of making changes to files directly on Github.  I'd rather have PRs for better grouping of changes.  But perhaps these PRs can be merged by the author.  (As per your point in the comment below).

Reply:
Explained in the previous comment thread.

Reply:
OK.